### PR TITLE
Update script.sh to remove the GPU drivers

### PR DESCRIPTION
**Pull request by vikasnav**

### CODE REVIEW

File: kubernetes/template/DeploymentTemplates/script.sh [hunk 1]
```diff
@@ -35,6 +36,20 @@ function collect_deployment_and_operations
 # Collect deployment logs always, even if the script ends with an error
 trap collect_deployment_and_operations EXIT
 
+###
+#   <summary>
+#       Cleanups up the GPU Drivers. This is needed when DVM is deployed with AKS Image that contains these drivers
+#   </summary>
+#   <returns>None</returns>
```
Comment (rjaini) [inline kubernetes/template/DeploymentTemplates/script.sh:43]: You need to make changes in azuredeploy.json as well for this change to apply.

Comment (rjaini) [reply kubernetes/template/DeploymentTemplates/script.sh:43]: https://github.com/msazurestackworkloads/azurestack-gallery/blob/master/kubernetes/template/DeploymentTemplates/azuredeploy.json#L430

Comment (vikasnav) [reply kubernetes/template/DeploymentTemplates/script.sh:43]: Yes I need to understand how to do that :)

Comment (vikasnav) [reply kubernetes/template/DeploymentTemplates/script.sh:43]: I am guessing we have to convert the script to base64 and update.

Comment (rjaini) [reply kubernetes/template/DeploymentTemplates/script.sh:43]: yes.

**Pull request closed without merging**